libibumad/umad.h

```diff
@@ -54,6 +54,21 @@ typedef __be16 __attribute__((deprecated)) be16_t;
 typedef __be32 __attribute__((deprecated)) be32_t;
 typedef __be64 __attribute__((deprecated)) be64_t;
 
+typedef union _ib_gid
+{
+	uint8_t	raw[16];
+
+	struct {
+		__be64	prefix;
+		__be64	interface_id;
+	} unicast;
+
+	struct {
+		uint8_t	header[2];
+		uint8_t	raw_group_id[14];
+	} multicast;
+} ib_gid_t;
```

Comment:
@shefty: I will drop the typedef.
@weiny2: The kernel header files define struct ib_gid in <rdma/ib_verbs.h> and struct rxe_gid in <uapi/rdma/rdma_user_rxe.h>. I just noticed that there is already a struct ibv_gid definition in libibverbs/verbs.h. I will see whether I can reuse that definition.